### DISCUSSION TITLE
YUNCLOUD USB Wifi

### DISCUSSION THREAD

**Discussion by tuancuong92** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#136] [Show and tell]
Hi Chris! I just bought a YUNCLOUD USB Wifi 1900Mbps from Ali express. After disable SIP, the adapter working perfectly on my hackintosh Big Sur 11.5.2
Many thanks for your work 🙏

**Comment by chris1111** [comment]
Alwright  so its a Realtek :RTL8814AU.
I will add to the working list
thanks
